client/src/app/pages/sbom-list/sbom-list.tsx

```diff
@@ -283,6 +285,13 @@ export const SbomList: React.FC = () => {
                                   downloadSBOM(item.id, `${item.name}.json`);
                                 },
                               },
+                              {
+                                title: "Delete",
+                                onClick: async () => {
+                                  await deleteSBOMById(item.id);
+                                  await refetch();
+                                },
```

Comment:
There is no need to for onClick to be async because the two functions used are promises.
```
   onClick: () => {
        deleteSBOMById(item.id);
        refetch();
  },                            },
```

Reply:
don't we need to wait for the delete to complete before we refetch?

Reply:
We do but if you use a React query this will be taken care off.

Reply:
Do you mean a react mutation?  Because the refetch() being used here is a react Query method.